### DISCUSSION TITLE
LitMotion in npmjs

### DISCUSSION THREAD

**Discussion by lejeanf** [AnnulusGames/LitMotion#91] [Ideas]
Hey,

I wanted to thank you for making LitMotion. It is great.

I Have a few questions about specific use cases but before that I would like to ask you if you could add LitMotion to Npm so that we can add it as depedency?
For instance, if I make custom package and publish it to npm and use your git link as depedency it gives an error because the git method does not use SimVer, meaning I cannot automatically import LitMotion as I import my custom package.

It would be very very practical if you did set it up.
That bein said I would very much be happy to help you set it up.
Its mainly about setting a github action and adding some tokkens for allowing git to publish stuff to npmjs.

Here is the package I use LitMotion for: https://github.com/lejeanf/UniversalPlayer
its still wip but feel free to use it!

Best,
Jean-François

**Comment by nuskey8** [comment]
Hello. Thank you for using LitMotion!

In my opinion, there is no need to publish directly to npm. If you want to use a scoped registry, publish on OpenUPM is probably a good option. (It appears that LitMotion is already available in OpenUPM.)

**Reply by lejeanf** [reply]
Thanks for the quick reply! OpenUPM will do nicely, even better. I only needed a a scoped registry! Thx for the tip.
